CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.8)
+
+# set the project name
+project(hershensh)
+
+# specify the C++ standard
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+set(INCLUDE_DIR include)
+set(TEST_DIR tests)
+
+set(HEADER_FILES ${INCLUDE_DIR}/reader.h ${INCLUDE_DIR}/evaluate.h ${INCLUDE_DIR}/printer.h)
+
+# add the executable
+add_executable(hershensh hershensh.cpp ${HEADER_FILES})
```

Comment:
This works:
```suggestion
add_executable(hershensh hershensh.cpp)
```

Reply:
Yes it works.
I added the headerrs so that IDEs know which files to load - see for [example](https://www.jetbrains.com/help/clion/managing-cmake-project-files.html#add-new-files).

Reply:
I don't see them doing this in the example or explaining why this is necessary but I think the discussion is not worth the time.